spec/order_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'order'
+#
+describe Order do
+  let(:menu) { double(:menu, dishes: dishes)}
+  let(:messager) { double(:messager)}
+  let(:dishes) { {menuitem1: 1, menuitem2: 1.50 } }
+
+  subject(:order) { described_class.new(menu_instance: menu, messager: messager)}
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)